helpers/SystemHelper.php

```diff
@@ -29,14 +29,17 @@ class SystemHelper
      */
     public static function getFileUploadLimit()
     {
+        $limits = [
+            self::toBytes(ini_get('upload_max_filesize')),
+            self::toBytes(ini_get('post_max_size'))
+        ];
 
-        $max_upload = self::toBytes(ini_get('upload_max_filesize'));
-        $max_post = self::toBytes(ini_get('post_max_size'));
-        $memory_limit = self::toBytes(ini_get('memory_limit'));
-
-        $returnValue = min($max_upload, $max_post, $memory_limit);
+        if (($memory_limit = self::toBytes(ini_get('memory_limit'))) !== -1) {
```

Comment:
Please don't do value setting and value checking in the if!
Check value from the ini_get directly and in case it's not -1 just add the converted value to the limits

Reply:
Done.